modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -361,12 +361,58 @@ m|+++10m+++
 |Description
 a|A comma-separated list of endpoints that a server should contact in order to discover other cluster members.
 Typically, all members of the cluster, including the current server, should be specified in this list.
+The setting configures the endpoints for Discovery service V2.
```

Comment:
This description should be reflected in the codebase as well. They should fully match.

Reply:
This [PR](https://github.com/neo-technology/neo4j/pull/26075) changes the description in the main codebase to reflect these changes.